db/package.js

```diff
@@ -102,61 +102,96 @@ class PackageDbController {
   }
 
   async refreshOwnership ({ packages, registry, maintainerId }) {
+    // find all the packages in the DB that either are marked as maintained by me,
+    // or simply have the provided registry and a name that's in the list provided by the registry
     const existingPackages = await this.db.collection('packages').find({
       $or: [
         { name: { $in: packages } },
-        { owner: maintainerId }
+        { 'maintainers.maintainerId': maintainerId }
```

Comment:
i changed this to "userId" in the latest PR i made, wdyt?

Reply:
will update; i like the idea